src/Form/Type/BasePickerType.php

```diff
@@ -72,7 +72,7 @@ public function finishView(FormView $view, FormInterface $form, array $options)
         if (isset($options['date_format']) && is_string($options['date_format'])) {
             $format = $options['date_format'];
         } elseif (is_int($format)) {
-            $timeFormat = ($options['dp_pick_time']) ? DateTimeType::DEFAULT_TIME_FORMAT : \IntlDateFormatter::NONE;
+            $timeFormat = ($options['dp_pick_time']) ? ($options['dp_use_seconds'] ? DateTimeType::DEFAULT_TIME_FORMAT : \IntlDateFormatter::SHORT) : \IntlDateFormatter::NONE;
```

Comment:
This is confusing, I think 1 ternary should be the limit.

Reply:
is it just a syntax problem, or do you dislike the idea of using the dp_use_seconds option?
I imagined otherwise to add an option time_format which would be by default DateTimeType::DEFAULT_TIME_FORMAT but that we could change to \IntlDateFormatter::SHORT.
I didn't did it because of the dp_use_seconds option which was defined, but which did not work as is.

Reply:
I dislike 2 ternaries, and the line is too long.